src/charts/helpers/locale.js

```diff
@@ -0,0 +1,29 @@
+define(function (require) {
+    const {FORMAT_LOCALE_URL} = require('./constants');
+    const {formatDefaultLocale} = require('d3-format');
```

Comment:
There needs to be a test for this helper. Make sure that every use-case is covered.

Reply:
Just to provide some context, this PR in its current state is only to discuss the current approach. If we decide it is the way to go, i can add it to the other charts and update docs/tests